### DISCUSSION TITLE
Rollup running through button on page but not through schedule

### DISCUSSION THREAD

**Discussion by dgccstesting** [jamessimone/apex-rollup#533] [Q&A]
Hi all, 

I am working with a fairly complex Salesforce org, with a number of rollups running, we are experiencing problems running rollups as scheduled jobs ( via flow and via Apex scheduled jobs ). We have installed the apex rollups package as a replacement for DLRS as it looked to be very promising. 

Our rollups seem to consistently "fail" when triggered via any kind of scheduled method. 

The interesting thing is the exact same rollups, run perfectly fine as "full calculate" jobs and also if triggered via the button on the lightning page. It is purely scheduled jobs/flows that fail ( with the exact same criteria supplied in the flow ). 

Our flow is very simple, it literally just triggers the required rollups via CMDT method. Even more bizarre is that the exact same flow, if triggered via a button on the record, explicitly for the record, works fine, problems only occur when attempting to run as a scheduled job for a batch. 

Has anybody had any experience with trying to run scheduled rollups and have them fail via scheduled flow, but succeed via triggering direct from button ? ( Even better, has anybody found a resolution for this ).

**Comment by jamessimone** [comment]
Can you start a trace on the user the scheduled job runs as and see what gets generated, log-wise?

**Comment by Fdhomont** [comment]
Hi dgcctesting,

I'm having a similar issue running rollups via a CMDT scheduled flow.
Did you find any resolutions in the end?
